Sources/PointFree/Account/Invoices.swift

```diff
@@ -304,3 +305,26 @@ let invoiceView = View<(Stripe.Subscription, Database.User, Stripe.Invoice)> { s
       ])
     ])
 }
+
+private func extraInvoiceInfo(subscription: Stripe.Subscription) -> [Node] {
+  guard let extraInvoiceInfo = subscription.customer.right?.extraInvoiceInfo else { return [] }
+
+  let extraInvoiceInfoNodes = intersperse(Html.br)
+    <| extraInvoiceInfo
+      .components(separatedBy: CharacterSet.newlines)
+      .filter { !$0.isEmpty }
+      .map(Html.text)
```

Comment:
honestly computed this `extraInvoiceInfoNodes` was prob the toughest thing on the whole PR :/ all I wanted to do was split the newlines and insert `<br>` tags between em. I needed to add the `filter` in order to accommodate for `\r\n` type newlines. 

lemme know if you have some better ideas...